API.md

```diff
@@ -110,6 +111,15 @@ method. The plugin accepts the following optional registration options:
           Defaults to `5000` (5 seconds).
         - `maxConnectionsPerUser` - if specified, limits authenticated users to a maximum number of
           client connections. Requires the `index` option enabled. Defaults to `false`.
+        - `expiresAt` - a function with a signature `async function(auth)`, where `auth` is the
```

Comment:
Not sure it needs to be an `async` function. In most (all?) cases the expiry info should be available synchronously from the auth?